### PR TITLE
-- iOS SDK Documentation updated for IDWise 3.0.1

### PR DESCRIPTION
**Pull request by HafizAhsan1**
-- Initialize method signature updated
-- IDWise.startJourney() called in a sequential manner
-- Documentation changed for clientKey instead of apiKey and apiSecret.

### CODE REVIEW

**Review by HafizAhsan1 — COMMENTED** [review]
@sakher Thanks for doing grammatical corrections.

File: ios-sdk.md [hunk 1]
```diff
@@ -66,18 +66,16 @@ import IDWise
...
-            }
-            else {
+        IDWise.initialize(clientKey: "<YOUR_CLIENT_KEY>") { error in
                 // Deal with error here
-                print(isSucceeded ,error?.code , error?.message)
-            }
+                print(error?.message)
         }
+        
+        IDWise.startJourney(journeyDefinitionId: "<YOUR_CUSTOMER_ID>", referenceNumber: "<YOUR_REFERENCE_NO>", locale: "en", delegate: self)
```
Comment (sakher) [inline ios-sdk.md:77]: Can we unify param names in both Android and iOS?
@waqar-idwise and @HafizAhsan1 

For example, referenceNo vs referenceNumber (i remember we settled on referenceNo ), 
journeyDefinitionId vs journeyTemplateId (I remember we settled externally on journeyTemplateId but still send journeyDefinitionId in the API).

**Review by sakher — APPROVED** [review]
Small suggestions, but happy overall, please merge once you are happy @HafizAhsan1

File: ios-sdk.md (hunk 1 above)
Comment (HafizAhsan1) [reply ios-sdk.md:77]: @sakher  will keep them in mind for next release. This one is already live on CocoaPods so next time I will surely sync this.

**Pull request merged**